pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you started.
+
+| All Keywords                | Specification | Draft introduction | Removed | Changed                                                                                      |
```

Comment:
question: @jdesrosiers and @jviotti, based on its contents, shouldn't the "Specification" column be named "Vocabulary"?

Reply:
The content is consistent with its heading. "Core", "Validation", and "Hyper-Schema" are the names of the two main spec documents. It is confusing because those are also names of vocabularies.